packages/lisk-transactions/src/utils/validation/validation.js

```diff
@@ -44,8 +48,13 @@ export const validatePublicKeys = publicKeys =>
 	checkPublicKeysForDuplicates(publicKeys);
 
 export const validateKeysgroup = keysgroup => {
-	if (keysgroup.length === 0 || keysgroup.length > 16) {
-		throw new Error('Expected between 1 and 16 public keys in the keysgroup.');
+	if (
+		keysgroup.length < MULTISIGNATURE_MIN_SIGNATURES ||
+		keysgroup.length > MULTISIGNATURE_MAX_SIGNATURES
```

Comment:
@shuse2 There's a difference between the range allowed for number of signatures and the range allowed for keysgroup length, right?

Reply:
I think `signatures` is misleading. the name should reflect `keysgroup`